src/AutoHideTab.cpp

```diff
@@ -537,6 +542,37 @@ void CAutoHideTab::mouseMoveEvent(QMouseEvent* ev)
     Super::mouseMoveEvent(ev);
 }
 
+//============================================================================
+void CAutoHideTab::dragEnterEvent(QDragEnterEvent* ev) {
+    if (CDockManager::testAutoHideConfigFlag(CDockManager::AutoHideOpenOnDragHover)) {
+        if (!d->TimeSinceDragOver.isValid()) {
+            d->TimeSinceDragOver.restart();
+            ev->accept();
+        }
+        else if (d->TimeSinceDragOver.hasExpired(500)) {
+            ev->accept();
+        }
+    }
+}
+
+//============================================================================
+void CAutoHideTab::dragLeaveEvent(QDragLeaveEvent* ev) {
+    if (CDockManager::testAutoHideConfigFlag(CDockManager::AutoHideOpenOnDragHover)) {
+        d->TimeSinceDragOver.invalidate();
+        d->forwardEventToDockContainer(ev);
+    }
+}
+
+//============================================================================
+void CAutoHideTab::dragMoveEvent(QDragMoveEvent* ev) {
+    if (CDockManager::testAutoHideConfigFlag(CDockManager::AutoHideOpenOnDragHover)
```

Comment:
If the mouse is not moved over the tab, then this event will not occure and the auto hide container will never be shown.